### ISSUE TITLE
eslint-plugin-jsx-a11y & eslint-config-airbnb error

### ISSUE THREAD

**Issue by Hongbo-Miao** [Hongbo-Miao/hongbomiao.com#4671]
Got error:

> Plugin "jsx-a11y" was conflicted between ".eslintrc.js" and ".eslintrc.js » eslint-config-airbnb » /home/runner/work/hongbomiao.com/hongbomiao.com/node_modules/eslint-config-airbnb/rules/react-a11y.js".

If fully remove `eslint-plugin-jsx-a11y` package in package.json, otherwise will get error

> Failed to load plugin 'jsx-a11y' declared in '.eslintrc.js » eslint-config-airbnb » /home/runner/work/hongbomiao.com/hongbomiao.com/node_modules/eslint-config-airbnb/rules/react-a11y.js': Cannot find module 'eslint-plugin-jsx-a11y'

If only remove `jsx-a11y` in `eslintrc.js`, got first error again...

**Comment by Hongbo-Miao** [comment]
Fixed by https://github.com/Hongbo-Miao/hongbomiao.com/pull/4188/commits/6537a6d4486ab00382dc9a99980b8897a664a35a

**Comment by Hongbo-Miao** [comment]
Fixed by https://github.com/Hongbo-Miao/hongbomiao.com/pull/4670
